core/primitives/src/version.rs

```diff
@@ -233,10 +232,9 @@ impl ProtocolFeature {
             | ProtocolFeature::LimitContractLocals
             | ProtocolFeature::ChunkNodesCache
             | ProtocolFeature::LowerStorageKeyLimit => 53,
+            ProtocolFeature::AltBn128 => 54,
```

Comment:
55, and you’ll also need to increase STABLE_PROTOCOL_VERSION.

Reply:
Good catch! I see that now we "jump" over versions 52 and 54, in a sense that these versions won't have any protocol features associated with them. How does this happen? Intuitively it seems that, to make a protocol change, we *should* have `ProtocolFeature`, so that we can apply old logic for old protocol versions.

Reply:
54 is a network layer protocol change when @pompon0 added protobuf support so it doesn’t affect the chain itself.  We probably should decouple the two versions at some point.  Though with protobufs it might be easier not to worry about network layer protocol version that much.